user-guide/Advanced_Functionality/Security/DataMiner_hardening_guide.md

```diff
@@ -210,3 +210,336 @@ If you use on-premises databases, we recommend using a [Cassandra cluster and Op
 >
 > - If you do use Storage as a Service, Skyline will take care of protecting your data, making use of existing and secure storage solutions provided by [Microsoft Azure](https://learn.microsoft.com/en-us/azure/storage/common/storage-introduction).
 > - Previously, Elasticsearch was recommended as the on-premises indexing database instead of OpenSearch. For more information on why OpenSearch is now recommended instead, see [From Elasticsearch to OpenSearch to StaaS](https://community.dataminer.services/from-elasticsearch-to-opensearch-to-staas/).
+
+## Secure pre-installed DataMiner Virtual Hard Disk
```

Comment:
Does the user actually have to do something here, or is this table merely meant for reference? Generally, it's always better not to present reference information as a procedure, so if this is meant for reference, I'd prefer to move this information elsewhere (e.g. a separate page under "About DataMiner security") to avoid confusion.
Because of the size of the table, I'd also prefer to change the way this information is presented, because right now when you start scrolling a bit, it's really difficult to keep track of where you are in the table. For example:
![image](https://github.com/user-attachments/assets/08a4b5bf-303d-4234-88c2-9a26b0f3ab08)

Reply:
It is meant as reference. For the pre-installed DataMiner nothing has to be done. OK to move.

Reply:
OK, I'll move this info.